Maddori.Apple/Maddori.Apple/Screen/View/HomeViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  HomeViewController.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 2022/10/18.
+//
+
+import UIKit
+
+import SnapKit
+
+class HomeViewController: UIViewController {
+    
+    // MARK: - property
+    
+    private let firstLabel: KeywordLabel = {
+        let keyword = KeywordLabel()
+        keyword.keywordType = .defaultKeyword
+        keyword.text = "첫 djdkdkdk번째"
+        return keyword
+    }()
+    private let secondLabel: KeywordLabel = {
+        let keyword = KeywordLabel()
+        keyword.text = "첫 번째"
+        keyword.keywordType = .previewKeyword
+        return keyword
+    }()
+    
+    // MARK: - life cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configUI()
+        render()
+    }
+    
+    // MARK: - func
+    
+    private func configUI() {
+        view.backgroundColor = .black
+    }
+    
+    private func render() {
+        view.addSubview(firstLabel)
+        firstLabel.snp.makeConstraints {
+            $0.left.equalTo(view.safeAreaLayoutGuide).inset(24)
+            $0.top.equalTo(view.safeAreaLayoutGuide).inset(24)
+        }
+        view.addSubview(secondLabel)
```

Comment:
p4;
실례가 안된다면 한칸... 
![image](https://user-images.githubusercontent.com/78677571/196610217-820a0c48-ee56-4466-a8a0-02e2cbf7f92e.png)

```suggestion
        }
        
        view.addSubview(secondLabel)
```

Reply:
헿 이전에 하던 습관이 나와버렸군요 ㅎㅎ...
[d5e69be](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/18/commits/d5e69bec4e17b49194c625a3eb7991602461980c)